host/CMakeLists.txt

```diff
@@ -294,15 +294,6 @@ endif ()
 if (OE_SGX AND WIN32)
   # oedebugrt is accessed via a bridge on Win32 and need not be linked.
   list(APPEND PLATFORM_SDK_ONLY_SRC sgx/windows/debugrtbridge.c)
-
-  # __oe_dispatch_ocall function must have debug symbols for debuggers to
-  # stitch the ocall stack. Use /Z7 flag to embed the debugging information
-  # in the object file. When a host application is built, the linker will
-  # extract this debugging information and put it in the pdb for the host
-  # application. Using /Z7 avoids having to distribut oehost.pdb.
-  # Note: This is a temporary fix and debuggers will not have this requirement
-  # when ocall stack stitching is also done via the debugger contract.
-  set_source_files_properties(sgx/calls.c PROPERTIES COMPILE_FLAGS "/Z7")
```

Comment:
Can you confirm that if this line is removed, you can debug a debug-mode enclave that is loaded by a Release mode host app built using release mode oehost.
If it is debuggable, then it is OK to remove this line. 
If not debuggable, it is better to retain this line for Release mode.

Reply:
After removing the /z7 flag from sgx/calls.c, verified that host built with CMAKE_BUILD_TYPE=Release was able to debug an enclave built with CMAKE_BUILD_TYPE=Debug and RelWithDebInfo.